cmp/acdf40a4-bde1-4006-93ec-4e985848c10c/component.lp

```diff
@@ -0,0 +1,65 @@
+(librepcb_component acdf40a4-bde1-4006-93ec-4e985848c10c
+ (name "Nixie Tube Without Dot")
+ (description "A nixie tube with 10 digits but no dot.")
+ (keywords "nixie,tube")
+ (author "Danilo Bargen")
+ (version "0.1")
+ (created 2019-09-16T22:18:58Z)
+ (deprecated false)
+ (category 0d3a9561-9a14-4622-98b2-d787b8b7478c)
+ (schematic_only false)
+ (default_value "{{ PARTNUMBER or DEVICE or COMPONENT }}")
+ (prefix "DS")
+ (signal d4b73128-f7c4-4fbc-aefd-30c56b56aef2 (name "C5") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 0b83ed1d-2265-4d50-b2e0-492f50254f2d (name "C4") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 31511e82-222c-415b-b5b8-aea8eeb9d6aa (name "C6") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 490887a1-1d21-4e86-9197-15ebdc09e3ba (name "C2") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal a2192dad-4ab8-4d76-ad2f-23ae7c6fe526 (name "C3") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal dd154e13-182e-4420-a53e-649181b3747b (name "C7") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 32d186e9-8141-46a4-967d-778645c6b329 (name "C0") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 999e5f79-c96d-41a8-ba6d-eca202d702d8 (name "A") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
```

Comment:
I have no clue about nixie tubes, but the symbol looks like "A" is required to be connected, isn't it? If yes:

```suggestion
  (required true) (negated false) (clock false) (forced_net "")
```

Reply:
Yep, you're right!